portfolio/src/main/webapp/index.html

```diff
@@ -167,6 +167,7 @@ <h2 id = "comments" > Comments </h2>
     
     <p>Comments:</p>
     <ul id="msg-container"></ul>
+    <button onclick="deleteComments()">delete comments</button>
```

Comment:
I would put a todo here to say you'll hook this up to the `/delete-data` servlet, or I would actually hook it up now so that the `DeleteCommentServlet` class is actually being utilized.

Reply:
deleteComments( ) in the js file calls a function in the DeleteCommentServlet class

Reply:
Oh oops, sorry I missed that!